src/storage/index.ts

```diff
@@ -387,46 +386,54 @@ function handleSignedEncryptedContents(caller: UserSession, path: string, stored
   })
 }
 
+export interface GetFileUrlOptions {
+  /**
+   * The Blockstack ID to lookup for multi-player storage. 
+   * If not specified, the currently signed in username is used.
+   */
+  username?: string;
+  /**
+   * The app to lookup for multi-player storage - defaults to current origin. 
+   * @default `window.location.origin` 
+   * Only if available in the executing environment, otherwise `undefined`.
+   */
+  app?: string;
+  /**
+   * The URL to use for zonefile lookup. If falsey, this will use 
+   * the blockstack.js's getNameInfo function instead. 
+   */
+  zoneFileLookupURL?: string;
+}
 
 /**
  * Used to pass options to [[UserSession.getFile]]
  */
-export interface GetFileOptions {
-  decrypt?: boolean,
-  verify?: boolean,
-  username?: string | null,
-  app?: string | null,
-  zoneFileLookupURL?: string | null
+export interface GetFileOptions extends GetFileUrlOptions {
+  /**
+   * Try to decrypt the data with the app private key.
+   * @default true
+   */
+  decrypt?: boolean;
+  /**
+   * Whether the content should be verified, only to be used 
+   * when [[putFile]] was set to `sign = true`.
```

Comment:
[[UserSession.putFie]] 

The code as written sends the user to the deprecated method